Holovibes/sources/gui/windows/panels/info_panel.cc

```diff
@@ -4,10 +4,8 @@
 
 #include "info_panel.hh"
 #include "MainWindow.hh"
+#include "information_worker.hh"
```

Comment:
Useless ?

Reply:
Nope used in the init function. It's one of the last part of the UI directly connected with back. I am thinking about how to remove it but it will gladly accept any thought